packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -737,6 +743,14 @@ interface WatchOptions {
    * @default - nothing extra is appended to the User-Agent header
    */
   readonly extraUserAgent?: string;
+
+  /**
+   * Whether to show CloudWatch logs for hotswapped resources
+   * locally in the users terminal
+   *
+   * @default - false
+   */
+  readonly logs?: boolean;
```

Comment:
I think this name should be more verbose than this.

`traceLogs`, or something?

(The name of the CLI flag is fine)

Reply:
updated to `traceLogs`